### ISSUE TITLE
question about age coefficient (coef (age.logFC))

### ISSUE THREAD

**Issue by PolinaShichkova** [czbiohub/tabula-muris-senis#40]
Dear all,

Thank you for sharing the great data!

I have probably a naive question, as I'm not very familiar with MAST and the details of your methods, and I definitely don't want to mess up something by reinventing your methods.

In your great paper https://www.nature.com/articles/s41586-020-2496-1  it's written multiple times "age coefficient threshold of 0.005 (corresponding to an approximately 10%-fold change)". 

In the file facs.Brain_Non-Myeloid.neuron.gz from https://figshare.com/articles/dataset/tms_gene_data_rv1/12827615?file=27856758 I see "coef (age.logFC)", which is the "age coefficient" from the paper as far as I understand.

My questions: 
1) Which transformation is applied to go from the age coefficient 0.005 to 10% fold change? 
2) Does 10% fold change in this case mean 100%*(old - young)/young or 100%*(old/young)?
3) Which base of the log is implied throughout the paper when it's not specified and given just as log?

In other words, what I need is to get rid of the log scale in age coefficient, and since none of the common base choices produce 0.1  from 0.005, I thought I may be missing something about the methods.

Thank you very much for the clarifications!

Best regards,
Polina

**Comment by aopisco** [comment]
@martinjzhang can you help @PolinaShichkova

**Comment by martinjzhang** [comment]
Hi @PolinaShichkova 

Thanks for your interest in our work.  

We used a linear model of `log expr ~ age + covariates`, where `age` is in the unit of month, and the regression coefficient for age is in the unit of log expr per month. 

>Which transformation is applied to go from the age coefficient 0.005 to 10% fold change?

Old mice are around 24 months and the young mice are around 3 months. Then if the age coefficient is 0.005, the log fold change between 24m mice and 3m mice is 

log expr(24m) - log expr(3m) = (24-3) * 0.005 which is roughly 0.1

> Does 10% fold change in this case mean 100%(old - young)/young or 100%(old/young)?

It means log (old expr / young expr) is around 0.1. 

> Which base of the log is implied throughout the paper when it's not specified and given just as log?

We used the natural base as in Scanpy log1p transformation https://scanpy.readthedocs.io/en/stable/generated/scanpy.pp.log1p.html#scanpy.pp.log1p 

Please see the comparison aging gene paper for more details. https://elifesciences.org/articles/62293

**Comment by PolinaShichkova** [comment]
Thank you very much! Everything is absolutely clear now! I just didn't expect that (24-3) multiplication factor.

Thank you!
